sdk/spring/azure-spring-boot-starter-keyvault-certificates/src/main/java/com/azure/spring/security/keyvault/certificates/starter/KeyVaultCertificatesEnvironmentPostProcessor.java

```diff
@@ -27,11 +27,6 @@
     @Override
     public void postProcessEnvironment(ConfigurableEnvironment environment, SpringApplication application) {
 
-        if (environment.getProperty("azure.keyvault.uri") == null) {
```

Comment:
is there any side effect with this remove?

Reply:
No side effect, I think.